server/src/dependencies.php

```diff
@@ -18,9 +18,19 @@
 	return new App\AppErrorHandler( $logger );
 };
 
+$container['dbalCache'] = function ( $c ) {
+	return new Doctrine\Common\Cache\FilesystemCache( __DIR__ . '/../cache' );
```

Comment:
Instead of using the filesystem, would it be better to use Redis?
https://wikitech.wikimedia.org/wiki/Help:Toolforge#Redis

There's also an adapter for it:
https://github.com/doctrine/cache/blob/master/lib/Doctrine/Common/Cache/RedisCache.php

for local development you can use an official redis image (just use a similar version):
https://hub.docker.com/_/redis/

Reply:
Meh, there is only one instance of this running so I didn't think that adding an external dependency would be beneficial. I can change it if you want, and if you do we might be better off doing it in a different ticket

Reply:
I mean it's already running on the server so it's not really another dependency per-say... If Toolforge is using an SSD the writting to the disk over and over again is not going to be very good for it...  so it would be better/faster to cache it in memory I think. if it's just a standard HDD on Toolforge then that's fine.. but that's slower than you might expect.